packages/timeline-state-resolver/src/integrations/shotoku/index.ts

```diff
@@ -260,30 +141,32 @@ export class ShotokuDevice extends DeviceWithState<ShotokuDeviceState, DeviceOpt
 
 		return commands
 	}
-	private async _defaultCommandReceiver(
-		_time: number,
-		cmd: ShotokuCommand,
-		context: CommandContext,
-		timelineObjId: string
-	): Promise<any> {
-		const cwc: CommandWithContext = {
-			context: context,
-			command: cmd,
-			timelineObjId: timelineObjId,
-		}
-		this.emitDebug(cwc)
+	async sendCommand({ command, context, tlObjId }: ShotokuCommandWithContext): Promise<any> {
+		this.emit('debug', { command, context, tlObjId })
 
 		try {
 			if (this._shotoku.connected) {
-				this._shotoku.executeCommand(cmd).catch((e) => {
+				this._shotoku.executeCommand(command).catch((e) => {
```

Comment:
This needs to be awaited.
The rethrown error will uncaught, and will likely crash the thread